yt_dlp/extractor/wrestleuniverse.py

```diff
@@ -137,12 +142,12 @@ def decrypt(data):
         }, query=query, fatal=fatal)
         return api_json, decrypt
 
-    def _download_metadata(self, url, video_id, lang, props_key):
-        metadata = self._call_api(video_id, msg='metadata', query={'al': lang or 'ja'}, auth=False, fatal=False)
+    def _download_metadata(self, url, video_id, lang, props_keys):
+        metadata = self._call_api(video_id, msg='video information', query={'al': lang or 'ja'}, auth=False, fatal=False)
         if not metadata:
             webpage = self._download_webpage(url, video_id)
             nextjs_data = self._search_nextjs_data(webpage, video_id)
-            metadata = traverse_obj(nextjs_data, ('props', 'pageProps', props_key, {dict})) or {}
+            metadata = traverse_obj(nextjs_data, ('props', 'pageProps', *props_keys, {dict})) or {}
```

Comment:
```suggestion
            metadata = traverse_obj(nextjs_data, ('props', 'pageProps', *variadic(props_keys), {dict})) or {}
```

Reply:
To be correct, this would need to follow traversal structure, which means
```suggestion
            metadata = traverse_obj(nextjs_data, ('props', 'pageProps', *variadic(props_keys, (str, bytes, dict, set)), {dict})) or {}
```

Reply:
Just variadic's good enough imo. But 🤷